app/models/classe.rb

```diff
@@ -4,7 +4,7 @@ class Classe < ApplicationRecord
   belongs_to :establishment
   belongs_to :mef
 
-  has_many :schoolings, dependent: nil
+  has_many :schoolings, dependent: :destroy
```

Comment:
why not, mais quel rapport avec notre zip ?

Reply:
je pense que c'était dans la chaîne de nettoyage des DA mais je vais l'enlever parce que je n'en suis plus sûr